media.css

```diff
@@ -0,0 +1,446 @@
+@media (min-width: 768px) and (max-width: 1023px) {
+	.wrapper {
+		width: 768px;
+	}
+	.header {
+		grid-template-columns: 1fr 1fr;
+		/*margin-bottom: 33px;
+		padding-top: 24px;*/
+		padding-bottom: 0;
+		margin-bottom: 26px;
+	}
+	.header__menu,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)